src/main/java/org/datadog/jmxfetch/JMXAttribute.java

```diff
@@ -73,6 +74,23 @@
         this.defaultTagsList = sanitizeParameters(beanParametersList);
     }
 
+    /**
+     * Remove tags listed in the 'exclude_tags' list from configuration.
+     */
+    private void applyTagsBlackList() {
+        Filter include = this.matchingConf.getInclude();
+        if (include != null) {
+
+            for (String excludedTagName : include.getExcludeTags()) {
+                for (Iterator<String> it = this.defaultTagsList.iterator(); it.hasNext();) {
+                    if (it.next().startsWith(excludedTagName + ":")) {
+                        it.remove();
+                    }
```

Comment:
Instead of this double `for` loop, can we have the `getExcludeTags` method to return a `Set`, and only iterates over `defaultTagsList ` performing lookups?